apps/studio/components/interfaces/Home/AdvisorWidget.tsx

```diff
@@ -0,0 +1,328 @@
+  TabsContent_Shadcn_ as TabsContent,
+  TabsList_Shadcn_ as TabsList,
+  TabsTrigger_Shadcn_ as TabsTrigger,
+} from 'ui'
+import ShimmeringLoader from 'ui-patterns/ShimmeringLoader'
+
+interface SlowQuery {
+  rolname: string
+  avg_time: number
+  calls: number
+  query: string
+}
+
+export const AdvisorWidget = () => {
+  const { ref: projectRef } = useParams()
+  const [selectedTab, setSelectedTab] = useState<'security' | 'performance'>('security')
+  const { data: lints, isLoading: isLoadingLints } = useProjectLintsQuery({ projectRef })
+  const { data: slowestQueriesData, isLoading: isLoadingSlowestQueries } = useQueryPerformanceQuery(
+    {
+      preset: 'slowestExecutionTime',
+    }
+  )
+  const snap = useAiAssistantStateSnapshot()
+
+  const securityLints = useMemo(
+    () => (lints ?? []).filter((lint: Lint) => lint.categories.includes('SECURITY')),
+    [lints]
+  )
+  const performanceLints = useMemo(
+    () => (lints ?? []).filter((lint: Lint) => lint.categories.includes('PERFORMANCE')),
+    [lints]
+  )
+
+  const securityErrorCount = securityLints.filter(
+    (lint: Lint) => lint.level === LINTER_LEVELS.ERROR
+  ).length
+  const securityWarningCount = securityLints.filter(
+    (lint: Lint) => lint.level === LINTER_LEVELS.WARN
+  ).length
+  const performanceErrorCount = performanceLints.filter(
+    (lint: Lint) => lint.level === LINTER_LEVELS.ERROR
+  ).length
+  const performanceWarningCount = performanceLints.filter(
+    (lint: Lint) => lint.level === LINTER_LEVELS.WARN
+  ).length
+
+  const top5SlowestQueries = useMemo(
+    () => ((slowestQueriesData ?? []) as SlowQuery[]).slice(0, 5),
+    [slowestQueriesData]
+  )
+
+  const totalIssues =
+    securityErrorCount + securityWarningCount + performanceErrorCount + performanceWarningCount
+  const hasErrors = securityErrorCount > 0 || performanceErrorCount > 0
+  const hasWarnings = securityWarningCount > 0 || performanceWarningCount > 0
+
+  let titleContent: React.ReactNode
+
+  if (totalIssues === 0) {
+    titleContent = <h2 className="text-xl">No issues available</h2>
+  } else {
+    const issuesText = totalIssues === 1 ? 'issue' : 'issues'
+    const numberDisplay = totalIssues.toString()
+
+    let attentionClassName = ''
+    if (hasErrors) {
+      attentionClassName = 'text-destructive'
+    } else if (hasWarnings) {
+      attentionClassName = 'text-warning'
+    }
+
+    titleContent = (
+      <h2 className="text-xl">
+        {numberDisplay} {issuesText} need
+        {totalIssues === 1 ? 's' : ''} <span className={attentionClassName}>attention</span>
+      </h2>
+    )
+  }
+
+  const renderLintTabContent = (
+    title: string,
+    lints: Lint[],
+    errorCount: number,
+    warningCount: number,
+    isLoading: boolean
+  ) => {
+    const topIssues = lints
+      .filter((lint) => lint.level === LINTER_LEVELS.ERROR || lint.level === LINTER_LEVELS.WARN)
+      .sort((a, b) => (a.level === LINTER_LEVELS.ERROR ? -1 : 1))
+      .slice(0, 5)
```

Comment:
hmm any reason why we're only choosing the first 5? reckon we can just show all tbh